### ISSUE TITLE
No visual distinction between enabled and disabled dates in date-picker.

### ISSUE THREAD

**Issue by tanmayray** [vmware/clarity#4685]
Next month's dates are grayed out by default. If all of the current month's dates are disabled, using min, then there is no visual distinction between disabled and enabled dates.

Code snippet for reproduction: 
<input type="date" clrDate name="sdate" [(ngModel)]="scheduleDate" min="2020-06-01"/>

Current date is 28-May-2020
Min date is 01-June-2020
Date picker displays month of May by default where all dates are disabled.
Default view shows few dates from June which are enabled but grayed.

Expected behavior: There should be visual distinction between enabled and disabled dates without depending on user action.

_A clear and concise description of what you expected to happen._

Versions:

**App**
- Angular: 9.1.7
- Node: 12.x
- Clarity: 3.1.3

**Device:**

- Type: Dell Latitute
- OS: Windows 10
- Browser: Chrome

Attaching screenshot for reference
![Capture](https://user-images.githubusercontent.com/66057462/83135971-4f43b580-a104-11ea-9b32-d0384f9839ab.PNG)

**Comment by gnomeontherun** [comment]
This seems like a semantic debate about having to distinguish between the previous/next month dates and disabled dates. While they may be _active_ they are not part of the _active month_. There isn't a lot of wiggle room with the design to support visually distinct variations that also have enough contrast.

What is the real problem? I'm not sure this is really a concern, as changing to the active month shows the active months with enough distinction.

**Comment by dexfab** [comment]
The issue is that disabled and excluded (previous and next month) days have the same styling (i.e. opacity set at .4), yet excluded days are still select-able without changing months. A different styling for disabled and excluded would resolve this.
